.tekton/pipeline-build.yaml

```diff
@@ -451,3 +451,31 @@ spec:
 
           - name: kind
             value: task
+
+    - name: notify-failure
+      when:
+        - input: $(tasks.status)
+          operator: in
+          values: [ "Failed", "Completed" ]
```

Comment:
why is `"Completed"` a failure?

Reply:
A Completed status means [one or more PipelineTask skipped and rest successful](https://tekton.dev/docs/pipelines/pipelineruns/#monitoring-execution-status).